### PR TITLE
Fixes/clarifications for home-robot installation

### PR DESCRIPTION
**Pull request by hello-binit**
## Motivation and Context

This PR includes a variety of fixes/clarifications to the installation docs and scripts.

## Changelog

 - Fix for installing git lfs
 - Fix docs for setting env vars
 - Fix docs for workstation install initial steps
 - Add nvcc utility to conda env
 - Fix dep missing on pinocchio
 - Fix dep missing on torch_geometric
 - Update stretch_grasping docs

## How Has This Been Tested

By testing out each fix in this PR on my system when I encountered an exception or error.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [x] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: install_deps.sh [hunk 1]
```diff
@@ -32,16 +32,21 @@ esac
 
 echo ""
 echo "Ensure Git LFS is installed"
+wget https://github.com/git-lfs/git-lfs/releases/download/v3.4.0/git-lfs-linux-amd64-v3.4.0.tar.gz
```
Comment (cpaxton) [inline install_deps.sh:35]: doesn't apt install git-lfs work better?

File: src/home_robot/environment.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ dependencies:
   - pinocchio
   - pytorch=2.0.0  # Compatibility with pytorch3d
   - pytorch-cuda=11.8
+  - cudatoolkit-dev
```
Comment (cpaxton) [inline src/home_robot/environment.yml:16]: @alexsax do you think this would help with our install issues?

Comment (cpaxton) [reply src/home_robot/environment.yml:16]: cc @yvsriram @ykarmesh for updates to install + instructions

File: install_deps.sh (hunk 1 above)
Comment (yvsriram) [reply install_deps.sh:35]: People generally do not have permissions to run apt commands on clusters. But not sure why we need this in addition to `git lfs install`. Will `pip install git-lfs` work as well?

File: src/home_robot/environment.yml (hunk 2 above)
Comment (yvsriram) [reply src/home_robot/environment.yml:16]: We should try installing from scratch once.

File: install_deps.sh (hunk 1 above)
Comment (cpaxton) [reply install_deps.sh:35]: Good point about lacking `sudo`

You actually do need to install this to run it; `git lfs install` does some preconfiguration stuff but as far as i know doesnt actually download the release

File: src/home_robot/environment.yml (hunk 2 above)
Comment (cpaxton) [reply src/home_robot/environment.yml:16]: @yvsriram best way to test a clean install is to just run the sim evaluation right? i can do this when i am in lab today

Comment (yvsriram) [reply src/home_robot/environment.yml:16]: Yes, that would be the best way. Thanks!

Comment (cpaxton) [reply src/home_robot/environment.yml:16]: @hello-binit i think we decided we need to remove this right?

File: src/home_robot_hw/environment.yml [hunk 3]
```diff
@@ -16,6 +16,7 @@ dependencies:
   - pinocchio
   - pytorch=2.0.*
   - pytorch-cuda=11.8
+  - cudatoolkit-dev
```
Comment (cpaxton) [inline src/home_robot_hw/environment.yml:19]: same here?

**Review by cpaxton — CHANGES_REQUESTED** [review]
from our installation debugging session

**Comment by cpaxton** [comment]
Moving these changes into #435 and closing

**Pull request closed without merging**